style.css

```diff
@@ -0,0 +1,3 @@
+* {
```

Comment:
[OPTIONAL]
- [ ] As stated in the project requirements, You were supposed to use the class selector `.class-name` and not the general selector `*` to give some styles to your title `Hello Microverse`